src/hearings/containers/request-hearing/hearing-edit-summary/hearing-edit-summary.component.ts

```diff
@@ -388,6 +388,7 @@ export class HearingEditSummaryComponent extends RequestHearingPageFlow implemen
                 hearingChannelEmail: this.compareAndUpdateServiceHearingValues(party.individualDetails?.hearingChannelEmail, serviceParty.individualDetails?.hearingChannelEmail, AutoUpdateMode.PARTY),
                 hearingChannelPhone: this.compareAndUpdateServiceHearingValues(party.individualDetails?.hearingChannelPhone, serviceParty.individualDetails?.hearingChannelPhone, AutoUpdateMode.PARTY)
               },
+              // this is the issue here or should i say this is where the value is set
```

Comment:
Should this comment be checked in?

Reply:
nope i totally forgot it was there
will remove